src/app/user/login/page.tsx

```diff
@@ -16,23 +17,25 @@ export default function LoginPage(): JSX.Element {
     }
   }, [isAuthenticated, router]);
 
+  if (error) {
+    if (error.message === 'INVALID_SUBSCRIPTION') {
+      new AuthError(error.message, error.message).show();
```

Comment:
we want the login page to still render when we have these kind of errors that's why we don't throw and just call show